src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts

```diff
@@ -322,15 +327,15 @@ class ProcessExplorer {
 		if (isLocal) {
 			items.push({
 				label: localize('killProcess', "Kill Process"),
-				click() {
-					process.kill(pid, 'SIGTERM');
+				click: () => {
+					this.electronService.killProcess(pid, false);
```

Comment:
its more descriptive of the api if the second arg was the kill code rather than a boolean.

`this.electronService.killProcess(pid, 'SIGTERM');`

Reply:
I like that 👍